src/modules/esl-media/core/esl-media.less

```diff
@@ -2,6 +2,12 @@
 .esl-media-init(@tagName: esl-media) {
   @{tagName} {
     display: block;
+    position: absolute;
+    inset: 0;
+    width: 100%;
+    height: 100%;
+    container-type: size;
+    overflow: hidden;
```

Comment:
It will definetly break something on consumers side
It looks like there is a custom styles but not the element base